tests/ott_sysmetrics.py

```diff
@@ -0,0 +1,29 @@
+from ott_lib import project_path, trace_loop
+
+if __name__ == "__main__":
+    trace_loop(1)
+
+
+class SysMetricsOtelTest:
+    def requirements(self):
+        return (project_path(),)
+
+    def environment_variables(self):
+        return {
+            "OTEL_SERVICE_NAME": "my-svc",
+            "OTEL_METRICS_ENABLED": "true",
```

Comment:
I think it is `true` by default so this line could be removed. Especially in the light of https://github.com/signalfx/splunk-otel-python/pull/565#discussion_r1867782025

```suggestion
```